### PR TITLE
Workspaces 😳✌🏼😍

### PR DESCRIPTION
**Pull request by inderpreetsingh**
Screenshot 2020-11-03 at 5 46 33 PM
Screenshot 2020-11-03 at 5 46 43 PM

### CODE REVIEW

File: www/main/workspace-bar/components/WorkspaceBar.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import { remote } from 'electron';
+
+const { store } = remote.require('./app');
+
+const WorkspaceBar = () => {
+  const workspaces = ['Presenter', 'Single Screen'];
+  const defaultWsState = store.getUserPref('app.layout.presenter-view')
+    ? workspaces[0]
+    : workspaces[1];
```
Comment (tsingh777) [inline www/main/workspace-bar/components/WorkspaceBar.jsx:10]: we should make this more scalability.  `app.layout.presenter-view` won't work when we have more tabs. maybe we can transition to a new UserPrefKey?

File: www/main/workspace-bar/components/WorkspaceBar.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import { remote } from 'electron';
+
+const { store } = remote.require('./app');
+
+const WorkspaceBar = () => {
+  const workspaces = ['Presenter', 'Single Screen'];
```
Comment (tsingh777) [inline www/main/workspace-bar/components/WorkspaceBar.jsx:7]: Should be translated.

Comment (maneetpaul) [reply www/main/workspace-bar/components/WorkspaceBar.jsx:7]: Can we also change "Presenter" to "Presentation", and "Single Screen" to "Single Display"

File: www/main/workspace-bar/components/WorkspaceBar.jsx (hunk 1 above)
Comment (inderpreetsingh) [reply www/main/workspace-bar/components/WorkspaceBar.jsx:10]: We are changing the settings infrastructure, so will change this along with that PR.

**Review by tsingh777 — APPROVED** [review]

**Pull request merged**